tests/test_triton_server.py

```diff
@@ -100,23 +104,24 @@ def test_server_config(self):
 
             # Make sure parsed value is the one from dict, check type too
             test_value = CLI_TO_STRING_TEST_ARGS[arg]
-            self.assertEqual(
-                test_value,
-                type(test_value)(value),
-                msg=f"CLI string contained unknown value: {value}")
+            self.assertEqual(test_value,
+                             type(test_value)(value),
+                             msg=f"CLI string contained unknown value: {value}")
+
+    def test_create_server(self, gpus=None):
```

Comment:
Perhaps you can rename this function to `_test_create_server` so that it is not called by the `unittest` and then you can add another variant that says `test_create_server_all_gpu` and explicitly call it with `gpus=['all']`

Reply:
Updated.